xExtension-ImageProxy/README.md

```diff
@@ -4,10 +4,62 @@ This FreshRSS extension allows you to get rid of insecure content warnings or di
+
+* `scheme_include` (default: `0`): whether to include the scheme - `http*://` - in the proxied URL
+
+* `url_encode` (default: `1`): whether to URL-encode (RFC 3986) the proxied URL
+
 ## Proxy Settings
 
 By default this extension will use the [images.weserv.nl](https://images.weserv.nl) image caching and resizing proxy, but instead you can supply your own proxy URL in the settings. An example URL would look like ``https://images.example.com/?url=``.
 
-The source code for the images.weserv.nl proxy can be found at [github.com/andrieslouw/imagesweserv](https://github.com/andrieslouw/imagesweserv), but of course other methods are available. For example, in Apache you could [use `mod_rewrite` to set up a simple proxy](https://httpd.apache.org/docs/2.2/rewrite/proxy.html) and similar methods are available in nginx and lighttpd. Alternatively you could use a simple PHP script, [along these lines](https://github.com/Alexxz/Simple-php-proxy-script). Keep in mind that too simple a proxy could introduce security risks, which is why the default proxy processes the images.
+By ticking the `scheme_https` checkbox, you can also force the use of the proxy, even for images coming through an encrypted channel. This makes the server that hosts your FreshRSS instance the only point of entry for images, preventing your client from connecting directly to the RSS sources to recover them (which could be a privacy concern in extreme cases).
+
+The source code for the images.weserv.nl proxy can be found at [github.com/andrieslouw/imagesweserv](https://github.com/andrieslouw/imagesweserv), but of course other methods are available. For example, in Apache you could [use `mod_rewrite` to set up a simple proxy](#apache-configuration) and similar methods are available in nginx and lighttpd. Alternatively you could use a simple PHP script, [along these lines](https://github.com/Alexxz/Simple-php-proxy-script). Keep in mind that too simple a proxy could introduce security risks, which is why the default proxy processes the images.
+
+### Apache configuration
+
+In order to use Apache [mod_rewrite](https://httpd.apache.org/docs/current/mod/mod_rewrite.html), you will need to set the following settings:
+
+* `proxy_url` = **https://www.example.org/proxy/**
+
+* `scheme_include` = **1**
+
+* `url_encode` = **0**
+
+Along the following Apache configuration for the `www.example.org` virtual host:
 
-By ticking the dedicated checkbox, you can also force the use of the proxy, even for images coming through an encrypted channel. This makes the server that hosts your FreshRSS instance the only point of entry for images, preventing your client from connecting directly to the RSS sources to recover them (which could be a privacy concern in extreme cases).
+```
+  # WARNING: Multiple '/' in %{REQUEST_URI} are internally trimmed to a single one!
```

Comment:
Btw, the two spaces in front of everything seem a touch random?

Reply:
Sorry. Copy-paste from my test config (fixed) :-)